proto/IntervalEdit.proto

```diff
@@ -0,0 +1,26 @@
+/**
+   Allele represents a single contiguous change 
```

Comment:
Extra copy/paste? Maybe "Represents the definite (not fuzzy) interval at which a sequence feature occurs, using interbase coordinates."